src/ivoc/ivocmain.cpp

```diff
@@ -248,7 +248,7 @@ void mac_open_app(){
 #endif
 
 extern "C" {
-	int ivocmain(int, const char**, const char**);
+	int ivocmain(int, const char**, const char**, int start_session);
```

Comment:
This is a change to the <implicit> api. It could break 3rd party software that loads neuron. (An example is
inithoc.cpp although that  is part of this software and so you noticed it as well.) Anyway, I'm wondering if
what is needed here is that the old ivocmain becomes a wrapper for ivocmain1 and gives it a default value
for the start_session which is consistent with the prior version. @ohm314 @pramodk

Reply:
I was afraid you'd say that. I did consider adding a new interface but keeping the old one in tact but rejected it because I was hoping to keep things simple. But, since this function part of the API, I will do as you suggested. Thank you for your feedback @nrnhines

Reply:
Is there some documentation of what is public interface and what isn't? If not, that's something to work on.

Reply:
That is definitely needed.